### ISSUE TITLE
Can't make this work on Windows XP

### ISSUE THREAD

**Issue by bjb43** [pachuco/dsoal-XP#1]
Hi, first let me thank you for forking dsoal for winXP! I can use your provided binary without issues under wine (set to WinXP), however when I try it on actual WindowsXP SP3, the applications (in my case Rightmark3DSound and MaxPayne) fail to start (the logs indicate they can't find sound devices/enumeration). I've checked that my registry is set up correctly. Would you have any more hints for me as to what I can try to make it run under XP? Thanks a lot!

**Comment by t9999clint** [comment]
I've had mixed results so far...  
sadly I don't have a copy of Max Payne at the moment to test with.

Works  
\-----------  
Serious Sam  
Serious Sam 2  
Star Wars Rouge Squadron - Can't actually tell if it's uses EAX  
Star Wars Battlefront - Set to 7.1 audio first before you copy over the dlls  
Unreal Tournament  
Unreal Tournament 2004  
FarCry  
MDK - A3D version works with A3D wrapper

Doesn't Work  
\-----------  
Star Wars Battlefront II - CTD on launch  
Star Wars Republic Commando - Doesn't let you enable EAX  
MDK II - Doesn't let you enable EAX  
F.E.A.R. - Doesn't let you enable EAX  
GRID - Crashes on launch

**Comment by t9999clint** [comment]
also it looks like Max Payne's compatibility is a upstream issue with the main DSOAL fork.  
kcat#4

**Comment by pachuco** [comment]
Yeah, I definitely fucked up something somewhere and updated without checking.
Stay tuned for further developments!

**Comment by pachuco** [comment]
[dsound.zip](https://github.com/pachuco/dsoal-XP/files/5744852/dsound.zip)

Try this dll.
Should fix "err:dsound:lazyLoad Couldn't load dsound: 998"

I am under suspicion that the release dll triggers error for some and not others.

**Comment by bjb43** [comment]
Thank you so much! With that dll, I can run e.g. RightMark 3DSound under WinXP without problems :), so that indicates to me that it is indeed working fine. I haven't tested many games so far, but I believe CS 1.5, MoHaa, JediKnight II work flawlessly! Also, I have no delay, which is great!

I'll test more games over the next days. Any particular games or tests I can help with?

**Comment by pappyN4** [comment]
This also fixes audio glitch in Star Trek Armada under XP.  

Previously I had to change the hardware acceleration in windows to "Basic" before playing to fix it, otherwise for speech during cut-scenes or ingame voices, it dropped words and squished the speech together.  But with this in the gamedir it worked fine with XP64
